cadc-data-ops-fits/src/test/java/org/opencadc/fits/slice/PolarizationCutoutTest.java

```diff
@@ -0,0 +1,226 @@
+        testHeader.addValue(Standard.CDELTn.n(4), -1.0D);
+        testHeader.addValue(Standard.CRPIXn.n(4), 1.0D);
+        testHeader.addValue(CADCExt.CUNITn.n(4), "");
+
+        final String[] states = new String[]{
+                PolarizationState.RR.name(),
+                PolarizationState.LL.name()
+        };
+
+        final PolarizationCutout testSubject = new PolarizationCutout(testHeader);
+
+        final long[] results = testSubject.getBounds(states);
+        final long[] expected = new long[]{1L, 300L, 1L, 300L, 1L, 151L, 3L, 4L};
+
+        assertFuzzyPixelArrayEquals("Wrong output.", expected, results);
+    }
+
+    @Test
+    public void testLinear() throws Exception {
+        final Header testHeader = new Header();
+
+        testHeader.addValue(Standard.NAXIS, 4);
+        testHeader.addValue(Standard.NAXISn.n(1), 300);
+        testHeader.addValue(Standard.CTYPEn.n(1), CoordTypeCode.RA.name() + "---SIN");
+        testHeader.addValue(Standard.CRVALn.n(1), 2.465333333333E+02D);
+        testHeader.addValue(Standard.CDELTn.n(1), -1.111111111111E-04D);
+        testHeader.addValue(Standard.CRPIXn.n(1), 1.510000000000E+02D);
+        testHeader.addValue(CADCExt.CUNITn.n(1), "deg");
+
+        testHeader.addValue(Standard.NAXISn.n(2), 300);
+        testHeader.addValue(Standard.CTYPEn.n(2), CoordTypeCode.DEC.name() + "--SIN");
+        testHeader.addValue(Standard.CRVALn.n(2), 2.434013888889E+01D);
+        testHeader.addValue(Standard.CDELTn.n(2), 1.111111111111E-04D);
+        testHeader.addValue(Standard.CRPIXn.n(2), 1.510000000000E+02D);
+        testHeader.addValue(CADCExt.CUNITn.n(2), "deg");
+
+        testHeader.addValue(Standard.NAXISn.n(3), 151);
+        testHeader.addValue(Standard.CTYPEn.n(3), CoordTypeCode.FREQ.name());
+        testHeader.addValue(Standard.CRVALn.n(3), 1.152750450330E+11D);
+        testHeader.addValue(Standard.CDELTn.n(3), -7.690066705322E+04D);
+        testHeader.addValue(Standard.CRPIXn.n(3), 1.000000000000E+00D);
+        testHeader.addValue(CADCExt.CUNITn.n(3), "Hz");
+
+        testHeader.addValue(Standard.NAXISn.n(4), 1);
+        testHeader.addValue(Standard.CTYPEn.n(4), CoordTypeCode.STOKES.name());
+        testHeader.addValue(Standard.CRVALn.n(4), 8.0D);
+        testHeader.addValue(Standard.CDELTn.n(4), 1.0D);
+        testHeader.addValue(Standard.CRPIXn.n(4), 1.0D);
+        testHeader.addValue(CADCExt.CUNITn.n(4), "");
+
+        final String[] states = new String[] {
+                PolarizationState.XX.name()
+        };
+
+        final PolarizationCutout testSubject = new PolarizationCutout(testHeader);
+
+        final long[] results = testSubject.getBounds(states);
+        final long[] expected = new long[]{1L, 300L, 1L, 300L, 1L, 151L, 1L, 1L};
+
+        assertFuzzyPixelArrayEquals("Wrong output.", expected, results);
+    }
+
+    @Test
+    @Ignore("Would this ever happen?")
+    public void testNoOverlap() throws Exception {
+
```

Comment:
Yes, you can have a cube with IQU on the polarization axis and the caller asks for RR. Simple test that should be included.